make.py

```diff
@@ -1023,11 +1032,24 @@ def register(ptype, **kwargs):
     aggregate_pot()
 
 elif command == "extract-po":
-    for addon in [file for file in glob.glob("*")
-                  if os.path.isdir(file)
-                  and file != "po"]:
-        print (addon)
+    check_gramps_path(command)
+    # add Gramps po to each of the base addon po creating "*_g.po" files
+    for g_po in glob.glob("*.po", root_dir=GRAMPSPATH + "/po"):
+        lang = g_po[:-3]
+        if not os.path.exists(f"po/{lang}.po"):
+            print(f"No language file for {lang} in po/ ?")
+            continue
+        system(
+            f'msgcat --use-first po/{lang}.po {GRAMPSPATH}/po/{lang}.po -o "po/{lang}_g.po"'
```

Comment:
Why do you need to do this?

The [get_addon_translator](https://github.com/gramps-project/gramps/blob/master/gramps/gen/utils/grampslocale.py#L662) method returns a translator that looks in both the `gramps` and `addon` domains for translations.

Reply:
I have a script that reports the percentage of strings done for each addon/lang.  It was consistently showing less than 100%, which led me to investigate what was missing.  The Gramps strings.  I studied the make "update" code and realized that the new process was not including the Gramps strings, where the old process was.

I confess that I did not test to see if the addon was actually using the Gramps strings when an individual string was missing.  So I assumed that we needed to emulate the old process.  I'll run some more tests to make sure I understand all this, and then rework this PR.  When I have some more time (life still happens!).

Reply:
I've added a few unit tests in PR [#2023](https://github.com/gramps-project/gramps/pull/2023) just to check that the translator was working as expected.

It actually looks in `gramps.mo` first and then uses `addon.mo` as the fallback.  Any strings from Gramps core that end up in an `addon.mo` file will be ignored.